QualifiedTeachersApi/tests/QualifiedTeachersApi.Tests/Jobs/JobFixture.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Moq;
+using QualifiedTeachersApi.DataStore.Crm;
+using QualifiedTeachersApi.Jobs.Scheduling;
+using QualifiedTeachersApi.Services.GetAnIdentityApi;
+using QualifiedTeachersApi.Services.Notify;
+using Xunit;
+
+namespace QualifiedTeachersApi.Tests.Jobs;
+
+public class JobFixture : IAsyncLifetime
+{
+    public JobFixture(DbFixture dbFixture)
+    {
+        DbFixture = dbFixture;
+    }
+
+    public DbFixture DbFixture { get; }
+
+    public Mock<IDataverseAdapter> DataverseAdapter { get; } = new Mock<IDataverseAdapter>();
+
+    public Mock<IBackgroundJobScheduler> BackgroundJobScheduler { get; } = new Mock<IBackgroundJobScheduler>();
+
+    public Mock<INotificationSender> NotificationSender { get; } = new Mock<INotificationSender>();
+
+    public Mock<IGetAnIdentityApiClient> GetAnIdentityApiClient { get; } = new Mock<IGetAnIdentityApiClient>();
+
+    public TestableClock Clock { get; } = new TestableClock();
+
+    public async Task InitializeAsync()
+    {
+        await DbFixture.InitializeAsync();
```

Comment:
I think we should call this from `Startup` instead of here, like we do for `ApiFixture`.

Reply:
Amended to use DbFixture in tests and removed JobFixture altogether